### PR TITLE
feat(ClientUser): Add support for setting bot banner

### PR DESCRIPTION
**Pull request by MinerPL**
**Please describe the changes this PR makes and why it should be merged:**

- https://github.com/discord/discord-api-docs/pull/6710

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by Jiralite** [comment]
[This is not documented](https://discord.com/developers/docs/resources/user#modify-current-user). We only document... documented features.

**Comment by MinerPL** [comment]
Released a moment ago:
DDevs Post: https://discord.com/channels/613425648685547541/697138785317814292/1218248663869030521
Documentation PR: https://github.com/discord/discord-api-docs/pull/6710

File: packages/discord.js/src/structures/ClientUser.js [hunk 1]
```diff
@@ -95,6 +100,20 @@ class ClientUser extends User {
     return this.edit({ avatar });
   }
 
+  /**
+   * Sets the banner of the logged in client.
+   * @param {?(BufferResolvable|Base64Resolvable)} banner The new avatar
+   * @returns {Promise<ClientUser>}
+   * @example
+   * // Set avatar
```
Comment (Jiralite) [inline packages/discord.js/src/structures/ClientUser.js:108]:
```suggestion
   * // Set banner
```

File: packages/discord.js/src/structures/ClientUser.js [hunk 2]
```diff
@@ -95,6 +100,20 @@ class ClientUser extends User {
     return this.edit({ avatar });
   }
 
+  /**
+   * Sets the banner of the logged in client.
+   * @param {?(BufferResolvable|Base64Resolvable)} banner The new avatar
```
Comment (Jiralite) [inline packages/discord.js/src/structures/ClientUser.js:105]:
```suggestion
   * @param {?(BufferResolvable|Base64Resolvable)} banner The new banner
```

**Review by Jiralite — CHANGES_REQUESTED** [review]

File: packages/discord.js/typings/index.d.ts
Comment (almeidx) [inline packages/discord.js/typings/index.d.ts:1]: `banner` is missing in the `ClientUserEditOptions` interface

**Review by almeidx — DISMISSED** [review]

**Review by Jiralite — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**